Sitemap/Voter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * A chain as a decorator for all tagged voter services.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Voter extends AbstractChain implements VoterInterface
```

Comment:
don't remember if we discussed this before, but could we remove the whole voting implementation and use our own instance of `AccessDecisionManager` with strategy `unanimous` and have the voters implement the symfony security `VoterInterface`?

i think the security component does a good job at modeling the whole voting process.